genesets/management/commands/genesets_add_geneset_tags.py

```diff
@@ -63,22 +110,33 @@ def handle(self, *args, **options):
             gs_id = toks[int(geneset_column)].split(':')
             gs_tag = toks[int(tag_column)]
             gs_name = toks[int(gs_name_column)]
-            gs_name = gs_name.replace('_', ' ') # Underscores may be used in files in place of spaces
-
-            if (len(gs_id) == 1): # If Id column was not able to be split in two by ':', state this term was not found
-                print('Geneset ' + str(toks[int(geneset_column)]) + '-' + str(gs_name) + ' and organism ' + str(org) + ' does not exist in our database')
+            # Underscores may be used in files in place of spaces
+            gs_name = gs_name.replace('_', ' ')
+
+            # If Id column was not able to be split in two by ':', state this
+            # term was not found
+            if (len(gs_id) == 1):
+                print('Geneset ' + str(toks[int(geneset_column)]) + '-' +
+                      str(gs_name) + ' and organism ' + str(org) +
+                      ' does not exist in our database'
+                )
```

Comment:
String formatting rather than concatenation would improve readability. What is the min python version required here?

Reply:
It should be fair to say Python 2.7. These management commands are only used in the tests right now, so I didn't spend too much time improving them.